recipes-devtools/cargo/cargo.inc

```diff
@@ -67,6 +67,10 @@ SRC_URI = "\
 
 B = "${S}"
 
+# Can't use the default from rust-vars as it cause a compiler crash
+# the crate_hash option will cause the compiler to crash
```

Comment:
That's news to me.  How are you ensuring that you're generating usable sstate entries without the crate_hash option?